src/amo/components/SecondaryHero/styles.scss

```diff
@@ -0,0 +1,72 @@
+@import '~amo/css/styles';
+
+.SecondaryHero {
+  color: $black;
+  display: flex;
+  justify-content: space-between;
+  min-height: 216px;
+  padding: $padding-page-l 0;
+}
+
+.SecondaryHero-message {
+  font-size: 16px;
+  line-height: 1.5;
+  margin-top: 16px;
+  width: 25%;
+}
+
+.SecondaryHero-message-headline {
+  font-size: $font-size-l;
+  line-height: 1.333;
+}
+
+.SecondaryHero-message-description {
+  padding: $padding-page 0;
+}
+
+.SecondaryHero-modules {
+  display: grid;
+  grid-gap: 32px;
+  grid-template-columns: repeat(3, 1fr);
+  width: 70%;
+}
+
+.SecondaryHero-module {
+  align-items: center;
+  background-color: $white;
+  border-radius: $border-radius-s;
+  display: flex;
+  flex-direction: column;
+  // flex-grow: 0;
+  font-size: 18px;
+  // justify-content: space-around;
+  line-height: 1.222;
+  // margin: 16px 0;
+  padding: 0 16px 16px 16px;
```

Comment:
Where did you get `16px` from? The padding does not seem consistent in the mocks but it's mostly around `22px` or `23px`. What about making it `24px`? That would match the spacing of some nearby elements.

Reply:
I think `16px` was a best guess to try to make it look good. As you said, in the mock the padding is inconsistent. I have changed it to 24px as you suggest.